### ISSUE TITLE
Incorrect file names in Duplicates Report

### ISSUE THREAD

**Issue by dnth** [visual-layer/fastdup#95]
I'm running on Ubuntu 20.04 and fastdup `0.214`.

I find that the file names in the duplicates report are not displayed correctly.

Steps to reproduce. 

```python
work_dir = "scene_classification/report/"
images_dir = "scene_classification/data/"

fd = fastdup.create(work_dir, images_dir)
fd.run()

fd.vis.duplicates_gallery() 
```
![image](https://user-images.githubusercontent.com/6821286/223345306-a46c3fcb-d37a-4edd-824f-244bea51354b.png)

I would also like it if the file names include the full directory. 
This is useful in the event we are visualizing duplicates that come from different folders.

The screenshot below shows the visualization with an older version of fastdup

![image](https://user-images.githubusercontent.com/6821286/223345933-cf3db2a8-dffe-4c0a-b53e-e51c6e7ac589.png)

**Comment by ovednagar** [comment]
Hi 
what you are seeing is the fastdup-id of the image, that was displayed instead of the image filename. 
fixed, should be in next version.

**Comment by dnth** [comment]
> Hi what you are seeing is the fastdup-id of the image, that was displayed instead of the image filename. fixed, should be in next version.

thank you @ovednagar !

**Comment by dbickson** [comment]
HI @dnth  Should be fixed in 0.901 please try it out and reopen if you still see an issue (better check all galleries)
